### DISCUSSION TITLE
Plans for reviewing product & services ontology

### DISCUSSION THREAD

**Discussion by NickeLind** [edmcouncil/fibo#1927] [Foundations (FND)]
Hello dear fibo community,

[The Q1 2023 release notes](https://github.com/edmcouncil/fibo/releases) mentions that the products and services ontology are being reviewed with the help of OMG’s Retail Domain Task Force, including GS1, with the IDMP team and with NIST’s Industrial Ontology Foundry (IOF) supply chain working group.

Are there plans for extending the ontology with more classes and properties for retail banking? Is the agreements ontology also being reviewed? Is there anyway to follow the progress of work and review sessions related to the product and services ontology?

This is something I am very interested in since we are in need of defining foundational patterns that allow us to build a flexible model for product classification including relationships with corresponding agreements within retail banking.

Regards,
Niklas Lind

**Comment by ElisaKendall** [comment]
@NickeLind 

Hi Niklas, 

My sincere apologies for not seeing this last summer when you posted it. The work on the Products and Services ontology is ongoing, and we hope to publish it as part of an update to the Commons Ontology Library, available at https://www.omg.org/spec/Commons, later this year.  That ontology will be general, though, not specific to Finance.  We will then update FIBO to use the new terminology and deprecate what is currently general in FIBO with respect to products and services. Some of the relevant updates include GTIN, UPC, the concept of a bar code and QR code, etc. that are all standard identifiers enabled by the GS1 standards.

We'd love to have you participate in the work on the addition of retail banking concepts in FIBO if you would like to do so. We have some upcoming work planned with FDIC, so it would be timely.  Please let me know if you are still interested and we can set up a time to touch base via webex.  

Best regards,

Elisa
